sa/precertificates.go

```diff
@@ -119,6 +120,38 @@ func (ssa *SQLStorageAuthority) AddPrecertificate(ctx context.Context, req *sapb
 	if overallError != nil {
 		return nil, overallError
 	}
+
+	// Store the OCSP response in Redis (if configured) on a best effort
```

Comment:
Sweet, looks to me like this code will be able to be copied (or factored out into a helper?) basically verbatim into SA.RevokeCertificate and SA.UpdateRevokedCertificate.